packages/order-utils/test/global_hooks_test.ts

```diff
@@ -0,0 +1,45 @@
+import { devConstants } from '@0xproject/dev-utils';
+import { ArtifactWriter } from '@0xproject/migrations';
+import { BigNumber } from '@0xproject/utils';
+
+import { artifacts } from '../src/artifacts';
+import { constants } from '../src/constants';
+import { DummyERC20TokenContract } from '../src/generated_contract_wrappers/dummy_e_r_c20_token';
+import { ERC20ProxyContract } from '../src/generated_contract_wrappers/e_r_c20_proxy';
+
+import { provider } from './utils/web3_wrapper';
+
+before('migrate contracts', async function(): Promise<void> {
+    // HACK: Since contract migrations take longer then our global mocha timeout limit
+    // we manually increase it for this before hook.
+    const mochaTestTimeoutMs = 20000;
+    this.timeout(mochaTestTimeoutMs);
+
+    const txDefaults = {
+        gas: devConstants.GAS_LIMIT,
+        from: devConstants.TESTRPC_FIRST_ADDRESS,
+    };
+
+    const networkId = constants.TESTRPC_NETWORK_ID;
+    const artifactsDir = `lib/src/artifacts`;
+    const artifactsWriter = new ArtifactWriter(artifactsDir, networkId);
+
+    const erc20proxy = await ERC20ProxyContract.deployFrom0xArtifactAsync(artifacts.ERC20Proxy, provider, txDefaults);
+    artifactsWriter.saveArtifact(erc20proxy);
```

Comment:
Why do we need to write those artifacts? Aren't those for tests?

Reply:
That's fair. I can imagine people writing their own migrations wanting access to this method... Any hesitations about exposing it?

Reply:
Good point was able to remove this by deploying within the test itself.